### PR TITLE
code fetch from sor skilldevelop branch

### PR DESCRIPTION
**Pull request by pitabash-eGov**

### CODE REVIEW

File: frontend/works_shg_app/lib/blocs/employee/emp_hrms/emp_hrms.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import 'dart:async';
+
+import 'package:dio/dio.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:works_shg_app/data/repositories/employee_repository/emp_hrms.dart';
+import 'package:works_shg_app/services/urls.dart';
+import 'package:works_shg_app/utils/global_variables.dart';
+
+import '../../../data/remote_client.dart';
```
Comment (Tulika-eGov) [inline frontend/works_shg_app/lib/blocs/employee/emp_hrms/emp_hrms.dart:12]: Why this file is added?

File: frontend/works_shg_app/lib/blocs/employee/mb/mb_crud.dart [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          //                         {
+          //                             "width": 0,
+          //                             "height": 0,
+          //                             "length": 0,
+          //                             "number": "2",
+          //                             "quantity": 2,
+          //                             "measurelineitemNo": 2
+          //                         },
+          //                         {
+          //                             "width": "1",
```
Comment (Tulika-eGov) [inline frontend/works_shg_app/lib/blocs/employee/mb/mb_crud.dart:97]: Remove comment please

File: frontend/works_shg_app/lib/data/repositories/employee_repository/work_order.dart [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  //       url,
+  //       queryParameters: queryParameters,
+  //       data: body ?? {},
+  //       options: Options(extra: {
+  //         "userInfo": GlobalVariables.userRequestModel,
+  //         "accessToken": GlobalVariables.authToken
+  //       }),
+  //     );
+
+  //     //return res.data;
```
Comment (Tulika-eGov) [inline frontend/works_shg_app/lib/data/repositories/employee_repository/work_order.dart:56]: Remove comment please

File: frontend/works_shg_app/lib/blocs/employee/emp_hrms/emp_hrms.dart (hunk 1 above)
Comment (pitabash-eGov) [reply frontend/works_shg_app/lib/blocs/employee/emp_hrms/emp_hrms.dart:12]: fetching the list of individual assignee . it is required in the MB confirmation screen

**Review by Tulika-eGov — APPROVED** [review]

**Pull request merged**